### DISCUSSION TITLE
[8 Giugno 2023] - Pipeline

### DISCUSSION THREAD

**Discussion by notedo** [CS-Swap/Architettura-degli-Elaboratori#21] [Pipeline]
### Traccia Esercizio

<img width="867" alt="img" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/f87d23f8-497f-4d00-a134-57d8ad786e31">
<img width="867" alt="img" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/58316afd-13c5-4696-9055-d4c86e689303">
<img width="864" alt="img" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/3d9be2a1-27dc-4714-9939-e92d8f747efd">

---

>**[Altre Soluzioni](https://github.com/sapienzastudentsnetwork/architettura-degli-elaboratori/discussions/84)**

**Comment by rimaout** [comment]
<img width="350" alt="B1AC7B54-EB37-420B-8F6F-50B02FFD989B" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/40bb20fc-e5c1-4423-8d3f-a4c3d446b011">

### 4) Senza Forwarding
<img width="835" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/cb8bd676-4c4b-44c1-97bf-b72570ec3cc8">

### 3 + 5) Con Forwarding
<img width="1052" alt="image" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/116072651/cf1ea58a-ef81-410c-96e1-c8286577745b">

### 6)
